gnat2goto/driver/goto_utils.ads

```diff
@@ -40,6 +40,10 @@ package GOTO_Utils is
    function Symbol_Expr (Sym : Symbol) return Irep
      with Post => Kind (Symbol_Expr'Result) = I_Symbol_Expr;
 
+   procedure New_Type_Symbol_Entry (Type_Name : String; Type_Type : Irep;
```

Comment:
The `Type_Name` is only used `Intern`'ed, so IMHO this should probably just use a `Symbol_Id`

Reply:
Intern was inlined at call-site.